Rubberduck.Inspections/Concrete/ObjectVariableNotSetInspection.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Diagnostics;
```

Comment:
I'm not sure I see where we actually use Diagnostics here...

Reply:
I had `Debug.Assert` calls in there at one point. I'll clean it up.